src/components/RadioPanel/RadioPanel.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode, Ref, forwardRef } from 'react'
+import { Panel, PanelBody, PanelContent } from '../Panel'
+import styled from '@emotion/styled'
+import { Radio, RadioProps } from '../RadioGroup/Radio'
+import { fontWeight, space } from '../../theme'
+import { Label } from '../Input'
+
+export interface RadioOption
+  extends Omit<RadioProps, 'id' | 'value' | 'label'> {
+  id: string
+  value: string
+  label: ReactNode
+  description?: () => ReactNode
+}
+
+export interface RadioPanelProps {
+  options: RadioOption[]
+  value: string
+  name: string
+  loading: string
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+const Wrapper = styled.span`
+  label {
+    display: grid;
+    grid-template-columns: ${space[24]} 1fr;
+    -webkit-align-items: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
```

Comment:
If I'm not mistaken the browser prefixes will be added automatically, of needed. I also don't see any other component do it.